### PR TITLE
Issue273/README fixes and HTML stats

### PR DESCRIPTION
**Pull request by mattloose**
This pull request addresses a number of issues (see commits).

All images are linked to the raw github urls to allow docs to propagate correctly.

### CODE REVIEW

**Comment by mattloose** [comment]
I also note I cannot spell the word brief.

File: README.md [hunk 1]
```diff
@@ -442,15 +446,121 @@ If you are happy with the speed of mapping, move on to testing a selection.
...
++----------------+----------+---------------+--------+----------+--------+---------------+-----------------+--------+--------------------+--------------------+-----------+------------+-----------+------------+----------+-----------+------------+----------+-----------+----------------+-----------+
+```
+The lower portion of the table shows the data broken down by contig in the reference (and so can be very long if using a complex reference!). 
+Again data are broken down by On and Off target. Read counts, yield, median and N50 read lengths are presented. 
+Finally we estimate the proportion of reads on target and an estimate of coverage.
+
+In this experiment, we were targeting chromosomes 20 and 21. As this is a playback run there is no effect on yield but you can see a clear effect on read length.
+The read length N40 and Median is higher for chromosomes 20 and 21 as expected. If running on more performant systems, the anticipated difference would be higher.
```
Comment (alexomics) [inline README.md:561]: An exotic N40 😯

Comment (mattloose) [reply README.md:561]: ROFL

**Comment by Adoni5** [comment]
# PR now covers two additions to readfish.

Closes #273 

## 1. HTML flag on stats
ReadfishSummarise 0.2.5 and by extension `readfish stats` outputs a HTML table of the summary via the use of an OPT IN flag `--html <FILENAME>`. This was added so we can embed a more readable table to the README.md

## 2. README updates
1. Readme images updated to Minknow 5.7.5 
1. Link to 5khz R10 NA12878 bulk file added
1. Links to the image sin the main branch of ReadFish added
1. Example analysis output from `readfish stats` using the `--html` flag added to the analysing test output section.

**Review by Adoni5 — APPROVED** [review]
Have been through the README, looks good. Once merged we should move refactor to main and have magic working images again.

**Pull request merged**